src/lib/operators/jit_operator/operators/jit_validate.cpp

```diff
@@ -18,6 +18,10 @@ bool is_row_visible(const CommitID our_tid, const TransactionID row_tid, const C
 
 JitValidate::JitValidate(const TableType input_table_type) : input_table_type(input_table_type) {}
 
+void JitValidate::before_specialization(const Table& in_table, std::vector<bool>& tuple_nullable_information) {
+  input_table_type = in_table.type();
```

Comment:
This shows a problem with the interface. Why do we pass in an unused parameter and do something different than in the other operators? Why do we overwrite the type that was explicitly set in the constructor?

Reply:
Brr..., yes. Also add a comment that the field will only be set correctly later.